package.json

```diff
@@ -105,7 +104,6 @@
     "typescript": "^4.1.3"
   },
   "dependencies": {
-    "@hapi/joi": "^17.1.1",
```

Comment:
Thanks @veksen - could you add `joi` and make sure the project still compiles? We also still need the types, unless the new version of `joi` comes bundled with them?

Reply:
The important part 😅 Added

Reply:
The types are now part of `joi`, if you want to double check that